.github/workflows/ci.yml

```diff
@@ -139,13 +139,14 @@ jobs:
       - name: Upload test report
         uses: actions/upload-artifact@v2
         # Always upload the test report for the annotate.yml workflow,
-        # but only the single XML file to keep the artifact small
+        # but only the single XML file and some logs to keep the artifact small
         if: always()
         with:
           # Name prefix is checked in the `Annotate checks` workflow
           name: test report ${{ github.job }}
           path: |
             **/surefire-reports/TEST-*.xml
+            /tmp/test-trino-*.log
```

Comment:
Log files tend to get very large, and if /tmp is mounted as tmpfs it is limited by RAM memory and not storage. Are there any guarantees on /tmp space avaialable on Github workers?